23.0.0.1-beta staging (#2824)
OpenLiberty/blogs: posts/2023-01-03-23.0.0.1-beta.adoc

---
layout: post
title: "Jakarta EE 10 Platform and MicroProfile 6"
# Do NOT change the categories section
categories: blog
author_picture: https://avatars3.githubusercontent.com/mbroz2
author_github: https://github.com/mbroz2
seo-title: Jakarta EE 10 Platform and MicroProfile 6 - OpenLiberty.io

> seo-description: "Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6. Also included in this beta release is a way to configure the maximum FFDC age and various bug fixes."

Reviewer: ```suggestion
seo-description: "Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6. This beta release also includes a way to configure the maximum age for FFDC files and various bug fixes."
```

> blog_description: "Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6. Also included in this beta release is a way to configure the maximum FFDC age and various bug fixes."

Reviewer: ```suggestion
blog_description: "Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6. This beta release also includes a way to configure the maximum age for FFDC files and various bug fixes."
```

open-graph-image: https://openliberty.io/img/twitter_card.jpg
open-graph-image-alt: Open Liberty Logo
---
= Jakarta EE 10 Platform and MicroProfile 6
Michal Broz <https://github.com/mbroz2>
:imagesdir: /
:url-prefix:
:url-about: /
//Blank line here is necessary before starting the body of the post.

> Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6.  To find out more, reference the link:{url-prefix}/blog/2022/12/06/22.0.0.13-beta.html[22.0.0.13-beta blog post].  Also included in this beta release is a way to configure the maximum FFDC age and various bug fixes about which you can learn more in the link:{url-prefix}/blog/2022/12/20/22.0.0.13.html#bugs[22.0.0.13 release blog post].

Reviewer: ```suggestion
Open Liberty 23.0.0.1-beta offers the same set of beta content as the previous 22.0.0.13-beta release: Jakarta EE Platform 10 and MicroProfile 6. To find out more, reference the link:{url-prefix}/blog/2022/12/06/22.0.0.13-beta.html[22.0.0.13-beta blog post]. This beta release also includes a way to configure the maximum age for FFDC files and various bug fixes, which you can learn more about in the link:{url-prefix}/blog/2022/12/20/22.0.0.13.html#bugs[22.0.0.13 release blog post].
```
